packages/pwa-buildpack/lib/WebpackTools/configureWebpack.js

```diff
@@ -291,26 +298,55 @@ async function configureWebpack({ context, vendor = [], special = {}, env }) {
             hints: 'warning'
         };
         config.devtool = false;
-        config.optimization.minimizer = [
-            new TerserPlugin({
-                parallel: true,
-                cache: true,
-                terserOptions: {
-                    ecma: 8,
-                    parse: {
-                        ecma: 8
-                    },
-                    compress: {
-                        drop_console: true
-                    },
-                    output: {
-                        ecma: 7,
-                        semicolons: false
-                    },
-                    keep_fnames: true
+        config.optimization = {
+            ...config.optimization,
+            moduleIds: 'hashed',
+            /**
+             * This will move the runtime configuration to
+             * its own bundle. Since runtime config tends to
+             * change on each compile even though the app logic
+             * doesn't, if not separated the whole client bundle
+             * needs to be downloaded. Separating them will only
+             * download runtime bundle and use the cached client code.
+             */
+            runtimeChunk: 'single',
```

Comment:
Is there any way to change the name of `runtime` ?  Since it's specific to webpack it'd be nice if it was `webpack-runtime` or something.

Reply:
Let me look into that. Not sure if that is possible.